VideoLocker/res/values/text_styles.xml

```diff
@@ -285,12 +285,20 @@
     </style>
 
     <!-- Discussion text styles -->
-    <style name="discussion_responses_small_text" parent="regular_edx_gray_base_text">
-        <item name="android:textSize">@dimen/edx_xxx_small</item>
+    <style name="discussion_regular_text" parent="regular_edx_gray_xdark_text">
+        <item name="android:textSize">@dimen/edx_small</item>
         <item name="android:layout_width">wrap_content</item>
         <item name="android:layout_height">wrap_content</item>
         <item name="android:gravity">center_vertical</item>

```

Comment:
As long as the layout width and height are defined as `wrap_content`, the gravity is non-applicable. Maybe this should be changed to `layout_gravity` instead?